Library/Homebrew/cmd/update-report.rb

```diff
@@ -695,6 +699,12 @@ def dump(updated_formula_report: true)
     puts
     puts <<~EOS
       You have #{msg} installed.
+    EOS
+    # If we're auto-updating, don't need to suggest commands that we're perhaps
+    # already running.
+    return if auto_update
+
+    puts <<~EOS
       You can upgrade #{update_pronoun} with #{Tty.bold}brew upgrade#{Tty.reset}
       or list #{update_pronoun} with #{Tty.bold}brew outdated#{Tty.reset}.
     EOS
```

Comment:
I agree that these messages really make the most sense when you're doing `brew update`. Fine with just returning early here.